pkg/maps/ipmasq/ipmasq.go

```diff
@@ -83,8 +82,10 @@ func IPMasq6Map() *bpf.Map {
 
 type IPMasqBPFMap struct{}
 
-func (*IPMasqBPFMap) Update(cidr net.IPNet) error {
-	if ip.IsIPv4(cidr.IP) {
+func (*IPMasqBPFMap) Update(cidr netip.Prefix) error {
+	cidr.Addr()
```

Comment:
leftover?

Reply:
Yikes, let me get rid of that.

Reply:
https://github.com/cilium/cilium/pull/34002